src/main/java/com/google/error_analyzer/backend/StoreLogs.java

```diff
@@ -108,4 +110,13 @@ public String getUniqueIndexName(String indexName) throws IOException {
         return nextIndexName;
     }
 
+    public void findErrorsInIndex(String indexName) {
+        try {
+            TimeUnit.SECONDS.sleep(1);
```

Comment:
why sleep?

Reply:
If the findErrorsInIndex is run right after storing the result is always 0 errors